src/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { Person } from './types';
+
+export const getPeopleWithParents = (people: Person[]) => {
+  return people.map(person => {
+    const mother = people.find(mom => person.motherName === mom.name || null);
+    const father = people.find(dad => person.fatherName === dad.name || null);
+    const motherName = !person.motherName ? '-' : person.motherName;
+    const fatherName = !person.fatherName ? '-' : person.fatherName;
```

Comment:
To avoid using magic values:
```suggestion
    const NOT_SET_VALUE = '-';
    ...
    const fatherName = !person.fatherName ? NOT_SET_VALUE : person.fatherName;
```